core/src/main/scala/cromwell/core/package.scala

```diff
@@ -10,4 +15,21 @@ package object core {
   type CallOutputs = Map[LocallyQualifiedName, JobOutput]
   type HostInputs = Map[String, WdlValue]
   type EvaluatedRuntimeAttributes = Map[String, WdlValue]
+  type StringEither[A] = Either[NonEmptyList[String], A]
+  type StringValidated[A] = ValidatedNel[String, A]
```

Comment:
If possible, I'd try to use the `ErrorOr[A]` from lenthall instead. Unless I'm misreading this, I think it does essentially the same thing, but it'd be more consistent with the rest of cromwell